tests/ut-torchapi.cc

```diff
@@ -308,9 +310,216 @@ TEST(torchapi, service_train_images)
                                     + iterations_resnet50 + ".pt"));
 
   fileops::clear_directory(resnet50_train_repo + "train.lmdb");
-  fileops::clear_directory(resnet50_train_repo + "test.lmdb");
+  fileops::clear_directory(resnet50_train_repo + "test_0.lmdb");
   fileops::remove_dir(resnet50_train_repo + "train.lmdb");
-  fileops::remove_dir(resnet50_train_repo + "test.lmdb");
+  fileops::remove_dir(resnet50_train_repo + "test_0.lmdb");
+}
+
+TEST(torchapi, service_train_images_multiple_testsets)
```

Comment:
I don't think it is useful to add other full training tests (with 200 iterations or so) just to check the "measure" field. Some assertions can be removed, and more specific assertions can be added. The training time can be reduced to 1 or 2 iterations, unless we want to check that the loss shrinks (but it is already done in other tests so it would be redundant here I think)

Reply:
you are very right, will do !

Reply:
done !